esmvaltool/diag_scripts/shared/iris_helpers.py

```diff
@@ -215,6 +245,29 @@ def intersect_dataset_coordinates(cubes):
     return new_cubes
 
 
+def preprocess_cube_before_merging(cube, cube_label):
```

Comment:
Maybe a different name without the word `preprocess` in it would be nicer, because this looks a bit like it's a preprocessor function. And maybe just the bit of the function that equalizes/strips all extra stuff is generic, while adding the label coord is not so generally useful?

Reply:
I will rename it to `prepare_cube_before_merging` and keep the function as it is, since it does exactly what the name suggests.